js/misc/eos-launch.js

```diff
@@ -0,0 +1,66 @@
+const Gio = imports.gi.Gio;
+const GLib = imports.gi.GLib;
+
+const AppLauncherIface = '<node> \
+<interface name="org.gnome.Shell.AppLauncher"> \
+<method name="Launch"> \
+    <arg type="s" direction="in" name="name" /> \
+    <arg type="u" direction="in" name="timestamp" /> \
+    <arg type="b" direction="out" name="success" /> \
```

Comment:
There should be no need to have a boolean field to indicate success in the interface, since DBus methods are failable already. Since this is a private interface between the helper and the shell, I think it's fine to change it.